docs/RFCS/README.md

```diff
@@ -9,15 +9,57 @@ Before making major changes, consider sending an RFC for discussion.
 This is a lightweight design-document process that is inspired by the
 [process used by the rust project](https://github.com/rust-lang/rfcs).
 
-1. Copy `template.md` to a new file and fill in the details.
-2. Submit a pull request to add your new file. Each RFC should get its
-   own pull request; do not combine RFCs with other files.
+1. Copy `template.md` or `template.rst` to a new file and fill in the
```

Comment:
Why do we need two different formats? Unless there is a good reason to introduce RST I'd rather standardize on markdown. Everyone's familiar with markdown through its use in github comments and other places, and markdown on github enjoys some extra features like auto-linking issue numbers.